src/js/_store/modules/categories.js

```diff
@@ -4,10 +4,33 @@ export default {
   namespaced: true,
   state: {
     categoriesList: [],
+    // メッセージをベタ書き
+    doneMessage: '',
+    notDoneMessage: '',
+    loading: false,
   },
   mutations: {
     getAllLists(state, res) {
-      state.categoriesList = res.categories;
+      state.categoriesList = res.categories.reverse();
+    },
+    getCategoryName(state, res) {
+      state.categoryPost = res.categories;
+    },
+    successMessage(state) {
+      state.doneMessage = 'カテゴリーが追加されました';
+    },
+    incompleteMessage(state) {
+      state.notDoneMessage = 'カテゴリーが追加できませんでした';
+    },
+    emptySuccessMessage(state) {
+      state.doneMessage = '';
+    },
+    emptyIncompleteMessage(state) {
+      state.notDoneMessage = '';
+    },
+    changeLoading(state) {
+      // 反転させる方法で
+      state.loading = (state.loading === 1) ? 0 : 1;
```

Comment:
[imo]

反転させるということをシンプルに記述すると以下のようになります。

```suggestion
      state.loading = !state.loading;
```

Reply:
state.loading = !state.loading;　に変更しました。